app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+end
+
+# A method that updates the posts counter for a user.
+def update_post_counter
+  User.select.update(:posts_counter).count
+end
+
+# if we add or delete a post from a user
+# the posts_counter will be updated in the user table
+
+# A method which returns the 5 most recent comments for a given post.
+def recent_5_comments
+  # Comments.last(5)
+  Comments.select('text').joins(:posts).last(5)
+end
```

Comment:
- [ ] These 2 methods should be inside the `Post` class
- [ ] I suggest that you use [rails active callbacks](https://guides.rubyonrails.org/active_record_callbacks.html) `after_save` to call the `update_post_counter` method every time a new post has been saved to the database, then make the `update_post_counter` method private

Reply:
Done.